.babelrc

```diff
@@ -1,9 +1,8 @@
 {
   "presets": [
-    ["env", {
+    ["@babel/env", {
       "modules": false
     }],
-    "stage-0",
```

Comment:
As best as I can tell this was used to allow for class properties.  Instead of bringing in all stage 0 proposals I brought in just the plugin for what was needed.